cub/util_type.cuh

```diff
@@ -96,19 +96,32 @@ using conditional_t = typename std::conditional<Test, T1, T2>::type;
 template <typename Iterator>
 using value_t = typename std::iterator_traits<Iterator>::value_type;
 
+template <typename It,
+          typename FallbackT,
+          bool = std::is_same<typename std::remove_cv<typename std::remove_pointer<It>::type>::type,
```

Comment:
Is there any particular reason not to use `::cuda::std::` instead of `std::` alternatives?

Reply:
No specific reason. Let's stick consistently to `::cuda::std`. Done 👍